### DISCUSSION TITLE
Cursor position when leaving insert mode

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#454] [Q&A]
Hi, 

How can I configure meow to move back the cursor position after leaving insert mode?

Similar to vis: 

https://github.com/martanne/vis/wiki/Differences-from-Vi(m)#cursor-positioning-when-leaving-insert-mode

**Comment by DogLooksGood** [comment]
I think you can write your own command for that, it's straightforward. However I don't think it's a good practice in Emacs modal editing.

**Reply by jgarte** [reply]
Oh ok,

So what you are saying is that I should just write a function that executes `left-char` after I get out of insert mode?

**Reply by DogLooksGood** [reply]
Yep. Actually, ESC doesn't move the cursor one character left in Vim. It's because the cursor is located on character in NORMAL mode, but between characters in INSERT mode. However, in Emacs, it's always between characters.
